apps/zotonic_mod_menu/src/mod_menu.erl

```diff
@@ -447,16 +461,18 @@ manage_schema(install, Context) ->
     #datamodel{
         categories = [
               {menu, categorization,
-               [{title, <<"Page menu">>}]
+                    #{
+                        <<"title">> => <<"Page menu">>
```

Comment:
I think that in English normally capitalisation is used for titles so "Page Menu"

Reply:
Changed